__tests__/app.test.js

```diff
@@ -44,3 +56,47 @@ describe("GET /api/categories", () => {
         .expect(404)
     })
 })
+
+
+describe("GET /api/reviews", () => {
+    test("Responds with an array of review objects, each with these properties: owner, title, review_id, category, review_img_url,  created_at, votes, designer, comment_count.", () => {
+    return request(app)
+        .get('/api/reviews')
+        .expect(200)
+        .then(( {body} ) => {
+            const {reviews} = body;
+            expect(reviews).toBeInstanceOf(Array);
+            expect(reviews).toHaveLength(13);
```

Comment:
Really like the idea of using a length check to ensure no false positives with empty values when looping over the reviews array below this.